pages/common/ts.md

```diff
@@ -0,0 +1,20 @@
+# ts
+
+> Add timestamps to input lines.
+> More information: <https://joeyh.name/code/moreutils/>.
+
+- Add current timestamp to beginning of each line:
+
+`{{some_command}} | ts`
+
+- Add timestamps with higher precision (microseconds):
+
+`{{some_command}} | ts "%b %d %H:%M:%.S"`
+
+- Add [i]ncremental timestamps, and show microseconds:
+
+`{{some_command}} | ts -i "%H:%M:%.S"`
+
+- Convert existing timestamps in log file into [r]elative format:
```

Comment:
I agree that the explanation should not be limited to "log files"; but I also think it would be good to mention that this feature is especially useful for log files. How about this:

```suggestion
- Convert existing timestamps in a text file (eg. a log file) into [r]elative format:
```

Or does that make the sentence too complicated? Or is it self-evident anyway that this is useful for log files?

Reply:
Good question. @CleanMachine1 what do you think?

Reply:
Sounds like a good idea!

Reply:
@oliver go for it then